joint_limits:
  joint_1:
    has_position_limits: true
    min_position: -1.0
    max_position: 1
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
    has_jerk_limits: true
    max_jerk: 100.0
  joint_2:
    has_position_limits: true
    min_position: -1.0
    max_position: 1
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
    has_jerk_limits: true
    max_jerk: 100.0
  joint_3:
    has_position_limits: true
    min_position: -1.0
    max_position: 1
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
    has_jerk_limits: true
    max_jerk: 100.0
  joint_4:
    has_position_limits: true
    min_position: -1.0
    max_position: 1
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
    has_jerk_limits: true
    max_jerk: 100.0
  joint_5:
    has_position_limits: true
    min_position: -1.0
    max_position: 1
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
    has_jerk_limits: true
    max_jerk: 100.0
  joint_6:
    has_position_limits: true
    min_position: -1.0
    max_position: 1
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 1.0
    has_jerk_limits: true
    max_jerk: 100.0